pkgs/tools/misc/kcollectd/default.nix

```
@@ -0,0 +1,52 @@
+{ lib
+, fetchFromGitLab
+, mkDerivation
+, fetchpatch
+, qtbase
+, cmake
+, kconfig
+, kio
+, kiconthemes
+, kxmlgui
+, ki18n
+, kguiaddons
+, extra-cmake-modules
+, boost
+, shared-mime-info
+, rrdtool
+}:
+
+mkDerivation rec {
+  pname = "kcollectd";
+  version = "0.11.99.0";
+  src = fetchFromGitLab {
+    owner = "aerusso";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0h4ymvzihzbmyv3z0bp28g94wxc6c7lgi3my0xbka3advxr811gn";
+  };
+
+  strictDeps = true;
+
+  nativeBuildInputs = [ cmake shared-mime-info ];
+
+  buildInputs = [
+    qtbase
+    extra-cmake-modules
+    kconfig
+    kio
+    kxmlgui
+    kiconthemes
+    ki18n
+    kguiaddons
+    boost
+    rrdtool
+  ];
+
+  meta = with lib; {
+    description = "A graphical frontend to collectd";
+    maintainers = [ maintainers.symphorien ];
+    license = [ lib.licenses.gpl3Plus ];
```

```suggestion
    license = [ lib.licenses.gpl3 ];
```

I only see https://gitlab.com/aerusso/kcollectd/-/blob/master/COPYING

---

see the header of any source file, they specify "or any later version"

---

i see, nevermind